data/Vaaran Bhai Gurdas Ji/28.json

```diff
@@ -1067,7 +1067,7 @@
         "id": "CVQU",
         "source_page": 28,
         "source_line": 6,
-        "gurmukhi": "gur isKI dw rwhu eyhu; gurmuiK cwl clY so dyKY [",
+        "gurmukhi": "guris`KI dw rwhu eyhu; gurmuiK cwl clY so dyKY [",
```

Comment:
No Adhak `gurisKI`

Reply:
Fixed. You'll need to open an issue or note somewhere that this needs to be explained in db why there is a difference.